cranelift/codegen/src/isa/aarch64/inst/mod.rs

```diff
@@ -2880,23 +2898,20 @@ impl ShowWithRRU for Inst {
                     }
                 }
             }
-            &Inst::OneWayCondBr {
-                ref target,
-                ref kind,
-            } => {
-                let target = target.show_rru(mb_rru);
+            &Inst::CondSkip { ref inst, ref kind } => {
+                let inst = inst.show_rru(mb_rru);
                 match kind {
                     &CondBrKind::Zero(reg) => {
                         let reg = reg.show_rru(mb_rru);
-                        format!("cbz {}, {}", reg, target)
+                        format!("cbz {}, 8 ; {}", reg, inst)
```

Comment:
uber nit, but this can be something else than 8 in the current form (say, if the embedded instruction requires generates many machine instructions). Maybe something like `cbz {}, $label ; { } ; $label:`?

(same thing a few times below)

Reply:
Keeping this now that it is just `TrapIf`, as the offset is now truly always `8`.